modules/gapi/include/opencv2/gapi/streaming/format.hpp

```diff
@@ -20,6 +20,19 @@ G_API_OP(GBGR, <GMat(GFrame)>, "org.opencv.streaming.BGR")
     static GMatDesc outMeta(const GFrameDesc& in) { return GMatDesc{CV_8U, 3, in.size}; }
 };
 
+G_API_OP(GY, <GMat(GFrame)>, "org.opencv.streaming.Y") {
+    static GMatDesc outMeta(const GFrameDesc frameDesc) {
+        return GMatDesc { CV_8U, 1, frameDesc.size , false };
+    }
+};
+
+G_API_OP(GUV, <GMat(GFrame)>, "org.opencv.streaming.UV") {
+    static GMatDesc outMeta(const GFrameDesc frameDesc) {
+        return GMatDesc { CV_8U, 2, cv::Size(frameDesc.size.width / 2, frameDesc.size.height / 2),
```

Comment:
frameDesc.size / 2 would be enough

Reply:
This doesn't work if I got in standalone mode, when frameDesc.size is cv::gapi::own::Size
But I will check once more time

Reply:
Yes, there is no operator overload for ``operator/`` for ``cv::gapi::own::Size``.
https://github.com/opencv/opencv/blob/master/modules/gapi/include/opencv2/gapi/own/types.hpp#L98